packages/alert/src/Alert.tsx

```diff
@@ -50,6 +58,26 @@ export interface AlertProps extends React.HTMLAttributes<HTMLElement> {
    * The variant to use. Defaults to "relative".
    */
   type?: 'relative' | 'toast' | 'banner'
+
+  /**
+   * Show / Hide close button
+   */
+  showCloseButton?: boolean
+
+  /**
+   * Show / Hide Alert
+   */
+  isOpen?: boolean
+
+  /**
+   * Show alert duration
+   */
+  delay?: number
+
+  /**
+   * Offset Index
+   */
+  offsetIndex?: number
```

Comment:
Is this for the stack offset? Can we make it more explicit by renaming it `stackIndex`,  although  it makes sense to keep using `offsetIndex` in `AlertWrapper`